src/js/poster-image.js

```diff
@@ -113,13 +101,32 @@ class PosterImage extends ClickableComponent {
   update(event) {
     const url = this.player().poster();
 
-    this.setSrc(url);
-
-    // If there's no poster source we should display:none on this component
-    // so it's not still clickable or right-clickable
     if (url) {
+      // As there's a poster source, add a picture/img if not already present
+
+      if (!this.$('img')) {
```

Comment:
this probably should move into `setSrc` so that either update or setSrc can be called without any errors. And it'll keep this method the same.